EpikCord/client/buckets.py

```diff
@@ -0,0 +1,181 @@
+from __future__ import annotations
+
+from ..utils import clear_none_values
+
+import asyncio
+from logging import getLogger
+from typing import Union, Optional, Dict
+
+import aiohttp
+
+logger = getLogger("EpikCord.http")
```

Comment:
why not using `__package__` or something like this for the logger?

Reply:
Isn't it `__name__`?

Reply:
Depends

Reply:
I want it to be static so that I can use one logger in multiple files.